packages/p2p-v2/src/pages/orders/components/ChatFooter/ChatFooter.tsx

```diff
@@ -20,18 +24,74 @@ const ChatFooter = ({ isClosed }: TChatFooterProps) => {
             </div>
         );
     }
+
+    const sendChatMessage = () => {
+        const elTarget = textInputRef.current;
+        const shouldRestoreFocus = document.activeElement === elTarget;
+
+        if (elTarget?.value) {
+            sendMessage(elTarget.value);
+            elTarget.value = '';
+            setValue('');
+
+            if (shouldRestoreFocus) {
+                elTarget.focus();
+            }
+        }
+    };
+
+    const handleKeyDown = (event: KeyboardEvent<HTMLInputElement>) => {
+        if (event.key === 'Enter' && !isMobile) {
+            if (event.ctrlKey || event.metaKey) {
+                const element = event.target as HTMLInputElement;
```

Comment:
I think you should use this here @nada-deriv, since the `element` we're interested in is the one that's raising and handling the `event`. 
```suggestion
                const element = event.currentTarget ;
```